extensions/ql-vscode/.storybook/preview.js

```diff
@@ -0,0 +1,38 @@
+import { themes } from '@storybook/theming';
+import { action } from '@storybook/addon-actions';
+
+// Allow all stories/components to use Codicons
+import '@vscode/codicons/dist/codicon.css';
+
+import '../src/stories/vscode-theme.css';
+
+export const parameters = {
+  actions: { argTypesRegex: "^on[A-Z].*" },
+  controls: {
+    matchers: {
+      color: /(background|color)$/i,
+      date: /Date$/,
+    },
+  },
+  docs: {
+    theme: themes.dark,
+  },
+  backgrounds: {
+    default: 'dark',
+    values: [
+      {
+        name: 'dark',
+        value: '#1e1e1e',
+      },
+    ],
+  },
+  options: {
+    storySort: {
+      order: ['WebView UI Toolkit', 'MRVA'],
```

Comment:
How come we have stories for the toolkit? It's not something we're building - it's an external dependency.

Reply:
This was how I started testing Storybook, but I agree with you that it doesn't really make sense to add stories for the WebView UI Toolkit.